ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -224,7 +224,7 @@ export default class ConfirmPageContainerContent extends Component {
                 className="actionable-message--warning"
                 message={
                   <Typography variant={TYPOGRAPHY.H7} align="left">
-                    {t('insufficientCurrency', [nativeCurrency, networkName])}
+                    {t('insufficientCurrency', [nativeCurrency, networkName])}{' '}
                     {t('buyOther', [nativeCurrency])}
```

Comment:
So currently for non mainnet networks we show this same message telling the user to `buy` the native currency of the network but we don't link them to any resource to do so:
![Screen Shot 2022-03-18 at 9 36 30 AM](https://user-images.githubusercontent.com/34557516/159023035-be115894-02d6-4ad4-ad86-b0a328fde620.png)

In cases like `MATIC` and `AVAX` we should probably link to the buy modal since we now support it. But for networks where we don't support a fiat onramp should we remove the "Buy __ or deposit from another account." for just "Deposit ___ from another account."? cc @rachelcope @SayaGT

Reply:
@adonesky1 yes that sounds correct. For the network that don't support a fiat onramp, just show "Deposit ___ from another account."

On separate note, can we activate **Buy** button for these networks without on ramp option. We can show "**Directly Deposit _Token_**". It basically let's user see their public account address. It will be a consistent experience across networks. What do you think @rachelcope 
<img width="349" alt="Screenshot 2022-03-18 at 15 01 52" src="https://user-images.githubusercontent.com/87030673/159028135-bba99a27-c73b-45f4-9371-b5d75dad904a.png">